### ISSUE TITLE
app-shell/push-3.4-r1 breaks eix

### ISSUE THREAD

**Issue by slackline** [vaeth/mv-overlay#34]
Related to #27 I upgraded to `app-shell/push-3.4-r1` yesterday and then `eix` reported..

```bash
 # eix-sync 
 * push.sh from https://github.com/vaeth/push/ is needed
 # eix app-shells/push
[U] app-shells/push
     Available versions:  3.4 3.4-r1(0/1)[1] {split-usr}
     Installed versions:  3.4(09:18:17 18/07/23)
     Homepage:            https://github.com/vaeth/push/
     Description:         A POSIX shell function to treat a variable like an array, quoting args

[1] "mv" /var/db/repos/mv
 # genlop -s app-shells/push
 * matches found:

     Mon Jul 17 07:03:08 2023 >>> app-shells/push-3.4-r1
 eix app-portage/eix
[I] app-portage/eix
     Available versions:  0.36.6 0.36.7^m[1] (~)0.36.7 **99999999*l[1] {debug +dep doc +jumbo-build +meson nls optimization +protobuf +required-use security sqlite +src-uri strong-optimization strong-security swap-remote tools usr-portage CPU_FLAGS_X86="sse2" L10N="de ru"}
     Installed versions:  0.36.7(09:51:24 18/07/23)(nls sqlite -debug -doc)
     Homepage:            https://github.com/vaeth/eix/
     Description:         Search and query ebuilds

[1] "mv" /var/db/repos/mv
```

If I downgrade to `app-shells/push-3.4` all is good again.

Apologies if I should have reported this in [vaeth/push](https://github.com/vaeth/push/) but since its the interplay between `eix` and `push` felt it fitted in here ok.

**Comment by vaeth** [comment]
Dupe of https://github.com/vaeth/eix/issues/63: The gentoo ebuilds for eix and push both install in a broken way (although they work with each other). The versions from the mv overlay both install correctly, but won't work with the broken installations from the gentoo ebuilds.

**Comment by slackline** [comment]
Sorry, that was even one that I reported and I read before posting.

Both Portage and the `mv` overlay have `app-portage/eix-0.36.7`, I've tried unmerging and re-emerging but it always pulls in the portage version rather than the one from `mv` which then aligns with `app-shells/push-3.4-r1` is there a way to force a package to be installed from an overlay if versions are the same? Is that why `-r1` is appended to `app-shells/push-3.4-r1`?
